### DISCUSSION TITLE
当添加element:click事件之后，如果legend点击取消了几个导致x轴变动，element:click带出来的数据就不对了

### DISCUSSION THREAD

**Discussion by 752134655** [antvis/G2#5800] [Q&A]
当添加element:click事件之后，如果legend点击取消了几个导致x轴变动，element:click带出来的数据就不对了，有什么好的解决办法嘛

**Comment by pearmini** [comment]
相同的问题，正在排查：https://github.com/antvis/G2/issues/5503
